packages/block-library/src/tag-cloud/edit.js

```diff
@@ -72,6 +72,6 @@ function TagCloudEdit( { attributes, setAttributes, taxonomies } ) {
 
 export default withSelect( ( select ) => {
 	return {
-		taxonomies: select( 'core' ).getTaxonomies(),
+		taxonomies: select( 'core' ).getTaxonomies( { per_page: -1 } ),
```

Comment:
per_page: -1 should be ignored as much as we can because it has the potential to trigger a high number of API requests. when the need for this comes up, we should consider whether to use a "ComboboxControl" or not. I'm not sure it's the case here though.

Reply:
This is for taxonomies, not terms of taxonomies. We use the same in the post sidebar, see https://github.com/WordPress/gutenberg/pull/9635/files. So I think it's okay to use it here, too. But you are right we should be careful with this parameter.